src/Form.module.css

```diff
@@ -0,0 +1,8 @@
+@import './theme/theme.css';
+
+.submitButton {
+    color: var(--color-main);
+    background: var(--color-secondary);
+    font-family: var(--font-main);
+    border: 2px solid red;
```

Comment:
Maybe red can also be used as a variable?

Reply:
This was just there as an example. When we decide what we want here we can change these variables.